src/library-authoring/add-content/messages.ts

```diff
@@ -6,6 +6,21 @@ const messages = defineMessages({
     defaultMessage: 'Collection',
     description: 'Content of button to create a Collection.',
   },
+  libraryContentButton: {
+    id: 'course-authoring.library-authoring.add-content.buttons.library-content',
+    defaultMessage: 'Existing Library Content',
+    description: 'Content of button to add existing library content to a collection.',
+  },
+  addToCollectionButton: {
+    id: 'course-authoring.library-authoring.add-content.buttons.library-content.add-to-collection',
+    defaultMessage: 'Add to Collection',
+    description: 'Button to add library content to a collection.',
+  },
+  selectedComponents: {
+    id: 'course-authoring.library-authoring.add-content.selected-components',
+    defaultMessage: '{count} Selected Component(s)',
+    description: 'Title for selected components in library.',
+  },
```

Comment:
nit: would be great if we could use a pluralized string here, but I wasn't able to work out how it should be formatted..

Reply:
react-intl has [great docs](https://formatjs.github.io/docs/react-intl/components#formattedmessage). Here is an example from our own codebase:

https://github.com/openedx/frontend-app-authoring/blob/11470f256d9ce330576c0771b5cbc0933dc3b51e/src/search-manager/messages.ts#L159-L163